src/openvpn/syshead.h

```diff
@@ -396,7 +396,7 @@
 /*
  * Do we have the capability to support the --passtos option?
  */
-#if defined(IPPROTO_IP) && defined(IP_TOS) && defined(HAVE_SETSOCKOPT)
+#if defined(IPPROTO_IP) && defined(IP_TOS) && defined(IPPROTO_IPV6) && defined(IPV6_TCLASS) && defined(HAVE_SETSOCKOPT)
```

Comment:
I initially though the same thing, but then I had a look at 86e2fa5597fd1ad8e0102f134c63d6bc8cb7c291 and thought not to handle that case. Do you think that on some platforms, IPv6 ToS might not be supported while IPv6 is?